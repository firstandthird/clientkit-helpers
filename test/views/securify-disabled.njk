{{ 'http://google.com' | securify }}
{{ 'https://yahoo.com' | securify }}
