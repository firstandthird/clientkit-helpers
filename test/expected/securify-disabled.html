http://google.com
https://yahoo.com
